flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/queryConfig.scala

```diff
@@ -92,6 +100,23 @@ class StreamQueryConfig private[table] extends QueryConfig {
     this
   }
 
+  /**
+    * Specifies a deferred computation time for deferred computation.
```

Comment:
We should enrich this comment. What is it used for? What is the impact of this setting?